utils/ort/src/main/kotlin/storage/HttpFileStorage.kt

```diff
@@ -36,7 +36,7 @@ import org.ossreviewtoolkit.utils.ort.OkHttpClientHelper
 import org.ossreviewtoolkit.utils.ort.execute
 
 private const val HTTP_CLIENT_CONNECT_TIMEOUT_IN_SECONDS = 30L
-private const val HTTP_CLIENT_KEEP_ALIVE_DURATION_IN_SECONDS = 5 * 60L
+private const val HTTP_CLIENT_KEEP_ALIVE_DURATION_IN_SECONDS = 60L
```

Comment:
Ah, so you were dropping the multiplication *here*. That's not what I meant. I meant that the first commit should use `TimeUnit.MINUTES` and thus no `* 60L` in the constant.

Reply:
Also, there are typos in "observeed" and "preceeding". Sorry for not noticing earlier.

Reply:
> Ah, so you were dropping the multiplication _here_. That's not what I meant. I meant that the first commit should use `TimeUnit.MINUTES` and thus no `* 60L` in the constant.

I would go your proposed route to say it's not worth discussing any further if this still is an option: https://github.com/oss-review-toolkit/ort/pull/6275#discussion_r1056101815.